cfgov/jinja2/v1/_includes/organisms/mega-menu.html

```diff
@@ -0,0 +1,125 @@
+
+{# ==========================================================================
+
+   sub_nav()
+
+   ==========================================================================
+
+   Description:
+
+   Creates a mega menu sub menu content markup when given:
+
+   sub_nav_title:   A string used for the sub nav title.
+
+   sub_nav_landing: A string used for the url of the sub nav landing page.
+
+   sub_nav_items:   An array of tuples.
+
+   sub_nav_media:   A string composed of html markup.
+
+   ========================================================================== #}
+
+{% macro _sub_nav(sub_nav_title, sub_nav_landing, sub_nav_items, sub_nav_media) %}
+<div class="o-mega-menu_subcontent" aria-expanded="false">
+    <div class="o-mega-menu_subcontent-wrapper">
+        <div class="wrapper">
```

Comment:
`_subcontent` creates a cropped "window" into which the menu slides down into.
`_subcontent-wrapper` sets the full width size, background color, and bottom drop-shadow.
`wrapper` sets the max-width and padding for the content to contain it within the center of the screen.

It would be nice to combine the inner two wrappers, but since they have two different widths that may be tricky.